plugins/source/aws/resources/services/iot/streams.go

```diff
@@ -45,20 +46,21 @@ func fetchIotStreams(ctx context.Context, meta schema.ClientMeta, parent *schema
 		if err != nil {
 			return err
 		}
-		for _, s := range page.Streams {
-			// TODO: Handle resolution in parallel with PreResourceResolver
-			stream, err := svc.DescribeStream(ctx, &iot.DescribeStreamInput{
-				StreamId: s.StreamId,
-			}, func(options *iot.Options) {
-				options.Region = c.Region
```

Comment:
Don't we need this function to set the region?

Reply:
No, these manual settings of region are a relic that never got cleaned up.

`c.Region` is always the same region that is set in the service initialization